src/reducers/summaryReducer.js

```diff
@@ -17,7 +18,7 @@ export default (state = initialState, action) => {
     case SUMMARY_RECEIVED:
       return {
         ...state,
-        ...action.summaries,
+        data: { ...action.summaries },
```

Comment:
i was thinking we wanted a fresh slate here so that we don't keep around, say, Ohio's data when Michigan is selected which might get confusing when we start munging that data